src/EA.Weee.Web/Areas/Aatf/Views/ManageEvidenceNotes/Overview/_SearchBySubmittedDatesPartial.cshtml

```diff
@@ -6,19 +6,11 @@
   @Html.Hidden("activeOverviewDisplayOption", ViewData["activeOverviewDisplayOption"])
 
     <div class="govuk-grid-column-one-third govuk-!-padding-left-2 govuk-!-padding-right-3">
-        <div class="govuk-form-group @Html.Gds().FormGroupClass(m => m.SubmittedDatesFilterViewModel.StartDate)">
-            @Html.Gds().LabelFor(m => m.SubmittedDatesFilterViewModel.StartDate, false)
+        <div class="govuk-form-group govuk-!-padding-top-20 @Html.Gds().FormGroupClass(m => m.SubmittedDatesFilterViewModel.StartDate)">
```

Comment:
That seems like a lot of padding, how does it look on screen?

Reply:
it seems that partial is currently unused.

Reply:
It should be this editor template SubmittedDatesFilterViewModel so it should be displayed on the AATF tab that has the filters

Reply:
As this has been UX reviewed can we remove the padding that has been added and then we can accept the PR @GuidoCangelosi